examples/evaluate_image_text_to_text_lmms_eval_inference.py

```diff
@@ -0,0 +1,36 @@
+from tqdm import tqdm
+from unitxt import settings
+from unitxt.api import evaluate, load_dataset
+from unitxt.inference import LMMSEvalInferenceEngine
+from unitxt.text_utils import print_dict
+
+with settings.context(
+    disable_hf_datasets_cache=False,
```

Comment:
We should document this new flag in the documentation in a relatively prominent way. (e.g. in one of the first tutorials that loads data from HF).

Reply:
Need to add this example to example main page, and disable the run of this examples  by default in regressions.